plugins/postcss-is-pseudo-class/test/compound-after-complex-is.css

```diff
@@ -0,0 +1,76 @@
+:is(.a .b).c {
+	order: 0.1;
+}
+
+.ignore :is(.a .b).c {
+	order: 0.2;
+}
+
+:is(.a .b).c .d {
+	order: 0.3;
+}
+
+:is(.a .b) .c {
+	order: 1.1;
+}
+
+.ignore :is(.a .b) .c {
+	order: 1.2;
+}
+
+:is(.a .b) .c .d {
+	order: 1.3;
+}
+
+:is(.a > .b) > .c {
+	order: 2.1;
+}
+
+.ignore > :is(.a > .b) > .c {
+	order: 2.2;
+}
+
+:is(.a > .b) > .c > .d {
+	order: 2.3;
+}
+
+:is(.a + .b) + .c {
+	order: 3.1;
+}
+
+.ignore + :is(.a + .b) + .c {
+	order: 3.2;
+}
+
+:is(.a + .b) + .c + .d {
+	order: 3.3;
+}
+
+:is(.a ~ .b) ~ .c {
+	order: 4.1;
+}
+
+.ignore ~ :is(.a ~ .b) ~ .c {
+	order: 4.2;
+}
+
+:is(.a ~ .b) ~ .c ~ .d {
+	order: 4.3;
+}
+
+.c:is(.a .b) .d {
+	order: 5.0;
+}
+
+c:is(a b) d {
+	order: 5.1;
+}
+
+/* https: //github.com/csstools/postcss-plugins/issues/922 */
+:is(.frame, .frame > .wrapper) > .target {
+  background-color: #ffc;
+}
+
+:is(:-csstools-matches(.a, .b)) > .c {
```

Comment:
I want to be sure that code looking for `:-csstools-matches` can also handle selector lists.

Internally we never produce `:-csstools-matches` with lists, but want to cover it anyway.

(We produce `:-csstools-matches` as an intermediate step with partially transpiled selectors to prevent infinite loops.)